sockeye/encoder.py

```diff
@@ -108,6 +131,18 @@ def get_rnn_cells(self) -> List[mx.rnn.BaseRNNCell]:
         """
         raise NotImplementedError()
 
+    def get_encoded_data_length(self, data_length: mx.sym.Symbol) -> mx.sym.Symbol:
+        """
+        Returns the data lengths of the encoded sequence.
+        """
+        return data_length
+
+    def get_encoded_seq_len(self, seq_len: int) -> int:
+        """
+        Returns the size of the encoded sequence.
+        """
+        return seq_len
+
```

Comment:
I think in the interest of not having too many dependencies between member functions that one needs to call, we should do this. We will have a backwards incompatible change with the configuration update anyway and need to bump the version. So this might be a good time for API changes as well.